### DISCUSSION TITLE
${file} obtaining path error

### DISCUSSION THREAD

**Discussion by SaboZhang** [dbeaver/dbeaver#19832] [General]
### Description

The path obtained by the variable is missing a slash after users, causing a path reading error. The normal path should be ```C:\users <username>\AppData\Local\Temp\dbeaver sql\format8385831020860966503SQL```, but the actual obtained path is ```C:\users<username>\AppData\Local\Temp\dbeaver\sql\format8385831020860966503SQL```
Attempting to switch workspaces and reinstalling both resulted in the same error

### DBeaver Version

CE 23.0.3.202304240616

### Operating System

Windows11

**Comment by SaboZhang** [comment]
should be ```C:\users\<username>\AppData\Local\Temp\dbeaver sql\format8385831020860966503SQL```

**Comment by SaboZhang** [comment]
![image](https://user-images.githubusercontent.com/34998007/235081543-1ec6f47d-c43a-4e10-a71c-1b9b17a72af9.png)

**Comment by LonwoLonwo** [comment]
Hello @SaboZhang 

Please provide steps to reproduce your issue.

**Comment by SaboZhang** [comment]
> 
1. Select SQL Editor in Preferences

2. Select SQL Format

3. Enter the directive at the command line ```node E : \\Documents\\Repository\\nodejs node_ global\\node_ modules\\sql-formatter\\bin\\sql-formatter-cli.cjs" -c config.json ${file}```

4. Check to use temporary file execution timeout

5. Click on the application prompt and the file cannot be found, followed by a path``` C: users<username> AppData Local Temp dbeaver sql format8385831020860966503SQL```
 thank you

**Comment by ShadelessFox** [comment]
Hello @SaboZhang,

Try surrounding `${file}` with quotes: `"${file}"`

**Comment by SaboZhang** [comment]
> Hello @SaboZhang,
> 
> Try surrounding `${file}` with quotes: `"${file}"`

Thank you for your suggestion. The issue has been successfully resolved
